cmd/node/factory/structs.go

```diff
@@ -588,7 +588,7 @@ func ProcessComponentsFactory(args *processComponentsFactoryArgs) (*Process, err
 	}, nil
 }
 
-func indexGenesisAccounts(accountsAdapter state.AccountsAdapter, indexer process.Indexer, marshalizer marshal.Marshalizer) error {
+func indexGenesisAccounts(startTime int64, accountsAdapter state.AccountsAdapter, indexer process.Indexer, marshalizer marshal.Marshalizer) error {
```

Comment:
_genesisTime_ instead of _startTime_ ?

Reply:
just kept the naming from `nodesSetup.json`